sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/SqlDelightEnvironment.kt

```diff
@@ -73,6 +74,10 @@ class SqlDelightEnvironment(
    * The package name to be used for the generated SqlDelightDatabase class.
    */
   private val properties: SqlDelightDatabaseProperties,
+  /**
+   * If true, fail migrations during compilation when there are errors.
+   */
+  private val verifyMigrations: Boolean,
```

Comment:
Can this be revisited and default to `true`. As far as I know that was the default behaviour

Reply:
with 1.4.3 its possible to have failing migrations (most people do because they don't have base `.db` files or `0.sqm`) so that would be a breaking change